.vib/kafka/goss/vars.yaml

```diff
@@ -0,0 +1,22 @@
+binaries:
+  - gosu
+  - java
+  - kafka-server-start.sh
```

Comment:
I recall we initially decided not to include these types of internal tools used at setup time, as having the container initialized (even though we are not doing so in the tests) would count as an implicit test.

Nevertheless, I agree we should just add every single binary added in the `bitnami_components.json` file to avoid confusion. I'll note updating the existing test suites down as a TODO.  FYI @mdhont, we talked about this a few days ago.